release/src/version-helpers.ts

```diff
@@ -94,8 +94,10 @@ export const getVersionFromReleaseBranch = (branch: string) => {
   return `v0.${majorVersion}.0`;
 };
 
+const SDK_TAG_REGEXP = /embedding-sdk-(0\.\d+\.\d+(-\w+)?)$/;
```

Comment:
We can drop `-nightly` now, since we have filtered out unsuppored prerelease IDs before this function is called.